### DISCUSSION TITLE
Pan image by dragging

### DISCUSSION THREAD

**Discussion by lubyant** [wkentaro/labelme#1165] [Ideas / Feature requests]
I just wonder if labelme can achieve this. Sometimes, my images are really big so I need to zoom in and trace the shape rigorously. In such case, I need to occasionally pan image. However, the way I found in labelme to do this is by clicking the scroll bar at the bottom and the right, which is somehow inconvenient. 
I found the software ArcMap has a really good implementation for this functionality and probably you can take it as a good example. 
<img width="200" alt="image" src="https://user-images.githubusercontent.com/71912657/187475175-f1d5bdc8-0ca3-491f-97b5-ea6f3ed0a388.png">

**Comment by ahiyantra** [comment]
I too faced similar issues with high-resolution images of microscopy slides. Having this feature would be great.
